### PR TITLE
[feat](Nereids) add lambda func `array_last` `array_first`

### PR DESCRIPTION
**Pull request by keanji-x**
## Proposed changes

add lambda func `array_last` `array_first`

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by XieJiann** [comment]
run buildall

**Comment by XieJiann** [comment]
run buildall

**Comment by XieJiann** [comment]
run p0

File: fe/fe-core/src/main/java/org/apache/doris/analysis/LambdaFunctionCallExpr.java [hunk 1]
```diff
@@ -45,6 +45,7 @@ public class LambdaFunctionCallExpr extends FunctionCallExpr {
     public static final ImmutableSet<String> LAMBDA_MAPPED_FUNCTION_SET = new ImmutableSortedSet.Builder(
             String.CASE_INSENSITIVE_ORDER).add("array_exists").add("array_sortby")
             .add("array_first_index").add("array_last_index").add("array_first").add("array_last").add("array_count")
+            .add("element_at")
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/analysis/LambdaFunctionCallExpr.java:48]: element_at is labmda function?

**Review by morrySnow — APPROVED** [review]

**Review by starocean999 — APPROVED** [review]

**Pull request merged**